assignment.js

```diff
@@ -113,6 +128,50 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const playerAChoice = "scissors"
+const playerBChoice = "scissors"
+
+if (playerAChoice === playerBChoice) {
```

Comment:
@annayensietopping nice work covering all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug.